tests/rkt_net_test.go

```diff
@@ -838,3 +845,113 @@ func NewTestNetLongName() testutils.Test {
 		testNetCustomNatConnectivity(t, nt)
 	})
 }
+
+func mockFlannelNetwork(t *testing.T, ctx *testutils.RktRunCtx) (string, networkTemplateT, error) {
+	// write fake flannel info
+	subnetPath := filepath.Join(ctx.DataDir(), "subnet.env")
+	file, err := os.Create(subnetPath)
+	if err != nil {
+		return "", networkTemplateT{}, err
+	}
+	mockedFlannel := fmt.Sprintf("FLANNEL_NETWORK=10.1.0.0/16\nFLANNEL_SUBNET=10.1.17.1/24\nFLANNEL_MTU=1472\nFLANNEL_IPMASQ=true\n")
+	_, err = file.WriteString(mockedFlannel)
```

Comment:
Since we don't need the first returned value, it's better to write:
`if _, err := file.WriteString(mockedFlannel); err != nil {`